core/lexer.mll

```diff
@@ -181,6 +181,7 @@ rule lex ctxt nl = parse
   | '_'                                 { UNDERSCORE }
   | '='                                 { EQ }
   | "->"                                { RARROW }
+  | "->>"                               { RRARROW }
```

Comment:
I don't want to introduce a new arrow. Let's remove this.

Reply:
Removed both from printing and parsing.